pages/_app.js

```diff
@@ -1,8 +1,25 @@
 import GlobalStyle from "../styles";
-import { shoppingItems } from "../lib/shoppingItemsData.js";
+import { shoppingItems as initialShoppingItems } from "../lib/shoppingItemsData.js";
 import { categories } from "../lib/categoriesData.js";
+import { useState } from "react";
+import { nanoid } from "nanoid";
 
 export default function App({ Component, pageProps }) {
+  const [shoppingItems, setShoppingItems] = useState(initialShoppingItems);
+
+  console.log("shoppingItems: ", shoppingItems);
```

Comment:
the console.logs in line 10 and 13 should be deleted

Reply:
Done. Thanks.